application/utils.py

```diff
@@ -0,0 +1,44 @@
+from constans.constans import EAST, NORTH, SOUTH, WEST
+from constans.constants_game import LARGE
+from game.board import Board
+from game.character import Character
+from game.player import Player
+
+
+def is_a_player_character(row, col, current_player: Player, board: Board) -> bool:
+    cell = board.get_cell(row, col)
+    character: Character = cell.character
+    return character.player.user_name == current_player.user_name if character else False
+
+
+def is_frendly_fire(row, col, direction, current_player: Player, board: Board) -> bool:
+    target_row, target_col = target_position_within_bounds(row, col, direction)
+    target_cell = board.get_cell(target_row, target_col)
+    if (
+        target_cell.character is not None and
+        target_cell.character.player.name == current_player.name
+    ):
+        return True
+    else:
+        return False
```

Comment:
NIT: unnecessary 'else' block. Consider an inline if:
`expression_if_true if condition else expression_if_false`
Or, directly:
```
    if (
        target_cell.character is not None and
        target_cell.character.player.name == current_player.name
    ):
        return True
       
    return False
```

Reply:
Great, I'm going to modify it!